docs/cpp/config-mingw.md

```diff
@@ -56,7 +56,7 @@ gdb --version
 
 From a Windows command prompt, create an empty folder called `projects` where you can place all your VS Code projects. Then create a sub-folder called `helloworld`, navigate into it, and open VS Code in that folder by entering the following commands:
 
-```cmd
+```bat
```

Comment:
shouldn't this be `bash` like in the file above rather than `bat`?

Reply:
The step says "from the windows command prompt"

Reply:
OK, got it. You were being precise in the context of the article.